help.html

```diff
@@ -275,35 +278,64 @@ <h2 id="hello">Hello, World!</h2>
         asking the micro:bit to scroll the words across the LED screen and then display an image of a heart.</p>
     <p>Why not try making the micro:bit scroll your name?</p>
    
-    <div class="hide-blocks">
-    <h2 id="blockly">Blockly</h2>
+    <div >
+    <h2 id="fs">File system</h2>
 
-    <p>Use blockly to make new Python code from easy to assemble blocks of
-    code.</p>
+    <p>Use the file system to add or remove python modules, data logs and other file types
+        from the micro:bit. Selecting the load button in the menu will bring up the files modal window.
```

Comment:
At this point it might be worth clarifying what the built-in modules are? "Add or remove extra modules" and find an appropriate place to find the built-in list at https://microbit-micropython.readthedocs.io/en/latest/py-modindex.html

Reply:
Have updated with links to built-in modules